src/__tests__/__snapshots__/fjs_babel-test.js.snap

```diff
@@ -721,6 +721,17 @@ var a: [number, string] = [123, \'duck\'];
 "
 `;
 
+exports[`test type-alias.js 1`] = `
+"type FBID = number;
+type Foo<T> = Bar<T>;
+type Maybe<T> = _Maybe<T, *>;
+
+type union = { type: \'A\' } | { type: \'B\' };
+
+type overloads = (x: string) => number & (x: number) => string;
```

Comment:
This is kinda broken, we should put parenthesis around the functions here, but i'm not sure where to put that logic, maybe in a more general helper like babel-generator does.